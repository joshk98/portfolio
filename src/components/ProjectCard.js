import React from "react";

import "../styles/projectcard.css";

function ProjectCard() {
  return <div>ProjectCard</div>;
}

export default ProjectCard;
