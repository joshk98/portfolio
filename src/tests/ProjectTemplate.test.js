import React from "react";
import { render } from "@testing-library/react";
import ProjectCard from "../components/projects/ProjectTemplate";

describe("ProjectCard", () => {
  it("renders correctly", () => {
    const { asFragment } = render(<ProjectCard />);

    expect(asFragment()).toMatchSnapshot();
  });
});
