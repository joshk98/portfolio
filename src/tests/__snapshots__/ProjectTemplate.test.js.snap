// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProjectTemplate renders correctly 1`] = `
<DocumentFragment>
  <div
    class="projectcard-container"
  >
    <ul
      class="projectcard-container-list"
    >
      <li
        class="projectcard-container-list-title"
      >
        Title
      </li>
      <li
        class="projectcard-container-list-type"
      >
        Type
      </li>
      <li
        class="projectcard-container-list-image"
      >
        Image
      </li>
      <li
        class="projectcard-container-list-description"
      >
        Description
      </li>
      <li
        class="projectcard-container-list-link"
      >
        Link
      </li>
    </ul>
  </div>
</DocumentFragment>
`;
